assets/css/_circle_button.scss

```
@@ -0,0 +1,29 @@
+@use "./color/tokens_2024" as tokens;
+
+.c-circle-button {
+  width: 2rem;
+  height: 2rem;
+
+  background-color: tokens.$gray-50;
+  border-radius: 50%;
+
+  &:hover {
+    background-color: tokens.$gray-100;
+  }
+
+  &:focus-visible {
+    border: 0.0625rem solid var(--bs-primary);
```

nit(non-blocking): just wondering if the border utilities would've worked here? https://getbootstrap.com/docs/5.3/utilities/borders/ seems like it would've been classes `border` and `border-primary`?

(comment originally authored at https://github.com/mbta/skate/commit/850d20296ea5d5dcc22375afcf1a7277c458f58a#r138593057 )

---

Though I guess this would've had to have been 
```suggestion
    @extend .border,.border-primary;
```
because of the `:focus-visible` state, unless bootstrap has like, "is focus visible" utilities that generate a `border-focus-visible` and `border-primary-focus-visible`

---

Yeah... maybe possible (I'd like to make it work) but seems like it might be too much effort for the moment.